tests/test_models.py

```diff
@@ -0,0 +1,69 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_string(self):
+        manufacturer = Manufacturer.objects.create(
+            name="Mazeratti",
+            country="Italy"
+        )
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_driver_string(self):
+        driver = get_user_model().objects.create(
+            username="anton_best",
+            password="anton12345",
+            first_name="Anton",
+            last_name="Best",
+        )
```

Comment:
In the `test_driver_string` method, the driver is created with a plain text password using `create`. It's recommended to use `create_user` to ensure the password is hashed correctly.